domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -72,7 +74,10 @@ class CameraXCameraUseCase @Inject constructor(
 
     //TODO apply flash from settings
     private val imageCaptureUseCase = ImageCapture.Builder().build()
-    private val previewUseCase = Preview.Builder().build()
+    private val previewUseCase: Preview = Preview.Builder().setResolutionSelector(
+        ResolutionSelector.Builder()
+            .setResolutionStrategy(ResolutionStrategy.HIGHEST_AVAILABLE_STRATEGY).build()
```

Comment:
Is there a reason we need to use `HIGHEST_AVAILABLE_STRATEGY`? I would think we would be better off relying on the default resolution.